tests/hats/io/test_parquet_metadata.py

```diff
@@ -64,6 +68,16 @@ def test_write_parquet_metadata_order1(
         small_sky_schema,
         0,
     )
+    ## the data thumbnail has 1 row group and a total of 4 rows
+    ## corresponding to the number of partitions
+    data_thumbnail_path = temp_path / "dataset" / "data_thumbnail.parquet"
+    assert data_thumbnail_path.exists()
+    thumbnail = ParquetFile(data_thumbnail_path)
+    data_thumbnail = thumbnail.read()
+    assert len(data_thumbnail) == 4
+    assert thumbnail.metadata.num_row_groups == 1
+    assert data_thumbnail.schema.equals(small_sky_schema)
+    assert data_thumbnail.equals(data_thumbnail.sort_by("_healpix_29"))
```

Comment:
```suggestion
    assert data_thumbnail.equals(data_thumbnail.sort_by(SPATIAL_INDEX_COLUMN))
```
Unless testing the value of the constant is part of the test?